UI/window-basic-settings.cpp

```diff
@@ -553,6 +553,7 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	HookWidget(ui->reconnectRetryDelay,  SCROLL_CHANGED, ADV_CHANGED);
 	HookWidget(ui->reconnectMaxRetries,  SCROLL_CHANGED, ADV_CHANGED);
 	HookWidget(ui->processPriority,      COMBO_CHANGED,  ADV_CHANGED);
+	HookWidget(ui->shutDownActiveOutputsOnExit, CHECK_CHANGED,  ADV_CHANGED);
```

Comment:
I was wondering about this one, is this okay or do you have a suggestion for this?